Modules/TOF/tofraw.json

```diff
@@ -61,6 +61,22 @@
           }
         ]
       },
+      "CheckDRMDiagnostics": {
+        "active": "true",
+        "className": "o2::quality_control_modules::tof::CheckDRMDiagnostics",
+        "moduleName": "QcTOF",
+        "detectorName": "TOF",
+        "policy": "OnAny",
+        "dataSource": [
+          {
+            "type": "Task",
+            "name": "TaskRaw",
+            "MOs": [
+              "RDHCounterCrate0"
```

Comment:
Idem, This should be the name of the histogram that you are processing in the checker